dash/dash-renderer/package.json

```diff
@@ -35,51 +35,51 @@
     "react": "16.14.0",
     "react-cytoscapejs": "^1.2.1",
     "react-dom": "16.14.0",
-    "react-json-tree": "^0.15.0",
-    "react-redux": "^7.2.5",
-    "redux": "^4.1.1",
+    "react-json-tree": "^0.15.1",
+    "react-redux": "^7.2.6",
+    "redux": "^4.1.2",
     "redux-actions": "^2.6.5",
-    "redux-thunk": "^2.3.0",
+    "redux-thunk": "^2.4.1",
     "regenerator-runtime": "^0.13.9"
   },
   "devDependencies": {
-    "@babel/cli": "^7.15.7",
-    "@babel/core": "^7.15.8",
-    "@babel/plugin-proposal-class-properties": "^7.14.5",
-    "@babel/preset-env": "^7.15.8",
-    "@babel/preset-react": "^7.14.5",
-    "@babel/preset-typescript": "^7.15.0",
-    "@plotly/eslint-config-dash": "0.0.1",
+    "@babel/cli": "^7.16.0",
+    "@babel/core": "^7.16.5",
+    "@babel/plugin-proposal-class-properties": "^7.16.5",
+    "@babel/preset-env": "^7.16.5",
+    "@babel/preset-react": "^7.16.5",
+    "@babel/preset-typescript": "^7.16.5",
+    "@plotly/eslint-config-dash": "0.0.2",
     "@plotly/prettier-config-dash": "0.0.1",
-    "@svgr/webpack": "^5.5.0",
-    "@types/chai": "^4.2.22",
+    "@svgr/webpack": "^6.1.2",
+    "@types/chai": "^4.3.0",
     "@types/mocha": "^9.0.0",
     "@types/ramda": "0.27.45",
     "@types/react": "^16.14.8",
     "@types/react-redux": "^7.1.20",
     "@types/redux": "^3.6.31",
     "@types/redux-actions": "^2.6.2",
-    "@typescript-eslint/eslint-plugin": "^4.28.1",
-    "@typescript-eslint/parser": "^4.28.1",
+    "@typescript-eslint/eslint-plugin": "^5.7.0",
+    "@typescript-eslint/parser": "^5.7.0",
     "babel-loader": "^8.2.3",
     "chai": "^4.3.4",
-    "css-loader": "^5.2.6",
-    "es-check": "^5.2.4",
-    "eslint": "^7.29.0",
+    "css-loader": "^6.5.1",
+    "es-check": "^6.1.1",
+    "eslint": "^8.4.1",
```

Comment:
So this `eslint` v8 is fine?

Reply:
oh thanks - yeah this one is also subject to the same danger. I'll revert it to v7.